.github/workflows/validate-terraform.yml

```
@@ -17,7 +17,7 @@ jobs:
     steps:
     - uses: hashicorp/setup-terraform@v1
       with:
```

Thanks @Chambras! Mind updating this to nix the `with:` version specification altogether since we no longer pin a specific Terraform version and instead `>=` in `src/terraform/*`?

```suggestion
    - uses: hashicorp/setup-terraform@v1
```

---

Done :)